modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc

```diff
@@ -8,10 +8,45 @@
 
 .Procedure 
 
-. Change the current directory to the front-end directory:
+ifeval::["{context}" == "creating-a-single-component-application-with-odo"]
+
+. Create a new directory for your components:
++
+----
+$ mkdir my_components $$ cd my_components
+----
+
+. Download the example Node.js application:
++
+----
+$ git clone https://github.com/openshift/nodejs-ex
+----
+endif::[]
+
+ifeval::["{context}" == "creating-a-multicomponent-application-with-odo"]
+
+. Download the example back-end and front-end applications:
++
+----
+$ git clone https://github.com/openshift-evangelists/Wild-West-Frontend frontend
```

Comment:
Does support know that you have instructions to clone content from github?

Reply:
Yes, we've approved all of our external URLs with GSS.